BUILD

```diff
@@ -38,3 +39,37 @@ cc_binary(
         "//conditions:default": ["as-nobody.c"],
     }),
 )
+
+
+# Docker images for buildfarm components
+java_image(
+    name = "buildfarm-server",
+    base = "@amazon_corretto_java_image_base//image",
```

Comment:
Why the corretto image base? ubuntu isn't satisfactory?

Reply:
@80degreeswest, do you know?

Reply:
We've been using it for about a year now. Here are the benefits: https://aws.amazon.com/corretto